pkg/command/login.go

```diff
@@ -4,31 +4,23 @@
 package command
 
 import (
+	"crypto/sha256"
+	"encoding/hex"
 	"fmt"
-	"os"
-	"sort"
 	"strings"
-	"time"
 
+	"github.com/google/uuid"
 	"github.com/spf13/cobra"
-	"golang.org/x/oauth2"
 
-	"github.com/vmware-tanzu/tanzu-plugin-runtime/component"
 	"github.com/vmware-tanzu/tanzu-plugin-runtime/config"
 	configtypes "github.com/vmware-tanzu/tanzu-plugin-runtime/config/types"
-	"github.com/vmware-tanzu/tanzu-plugin-runtime/log"
 	"github.com/vmware-tanzu/tanzu-plugin-runtime/plugin"
 
-	"github.com/vmware-tanzu/tanzu-cli/pkg/auth/csp"
-	tkgauth "github.com/vmware-tanzu/tanzu-cli/pkg/auth/tkg"
-	kubecfg "github.com/vmware-tanzu/tanzu-cli/pkg/auth/utils/kubeconfig"
 	"github.com/vmware-tanzu/tanzu-cli/pkg/cli"
-	"github.com/vmware-tanzu/tanzu-cli/pkg/pluginmanager"
+	"github.com/vmware-tanzu/tanzu-cli/pkg/utils"
 )
 
-var (
-	name, server string
-)
+var loginEndpoint string
 
 var loginCmd = &cobra.Command{
 	Use:     "login",
```

Comment:
After updating the docs for `login` command, I am more inclined to update this to `Login to Tanzu Application SaaS platform`. But as you mentioned, I am not sure if this would be future-proof?

@vuil Please let me know you thoughts.

Reply:
As per our internal discussion, I updated the short desciprtion to `Login to Tanzu Application Platform`.